src/commands/createNewProject/CSharpProjectCreator.ts

```diff
@@ -120,4 +124,28 @@ export class CSharpProjectCreator implements IProjectCreator {
     public getRecommendedExtensions(): string[] {
         return ['ms-vscode.csharp'];
     }
+
+    /**
+     * Validates the project has the minimum Functions SDK version that works on all OS's
+     * See this bug for more info: https://github.com/Microsoft/vscode-azurefunctions/issues/164
+     */
+    private async validateFuncSdkVersion(csprojPath: string, csprojContents: string): Promise<void> {
+        try {
+            const minVersion: string = '1.0.8';
+            const lineMatches: RegExpMatchArray | null = /^.*Microsoft\.NET\.Sdk\.Functions.*$/gm.exec(csprojContents);
+            if (lineMatches !== null && lineMatches.length > 0) {
+                const line: string = lineMatches[0];
+                const versionMatches: RegExpMatchArray | null = /Version="(.*)"/g.exec(line);
```

Comment:
Should you be more specific on the version regex?  e.g. Version=([0-9.]+) or something?

Reply:
I made it more specific, but not quite that specific (Sometimes they append things like "-beta" to the end)